scripts/participateInAuction.js

```diff
@@ -1,19 +1,25 @@
 /* eslint-disable no-console */
 const Coordinator = artifacts.require("Coordinator")
+const { Gastimator } = require("./gas_station")
+
+const GasStation = new Gastimator()
+
+// TODO - make this part of configuration
+const url = "https://safe-relay.rinkeby.gnosis.pm/api/v1/gas-station/"
 
 module.exports = async (callback) => {
   try {
     const coordinator = await Coordinator.deployed()
     console.log("Coordinator deployed at %s", coordinator.address)
     if (await coordinator.canParticipate()) {
-      await coordinator.participateInAuction()
+      const gasPriceEstimates = await GasStation.estimateGas(url)
```

Comment:
You might want to handle the failure case here. Otherwise our bot will fail if e.g. the gas endpoint is down.

Reply:
I kind of assumed this would be caught by the global try-catch in this file

Reply:
That is the case, however it will prevent the bot from triggering participate. Given that getting the right gas price is an optimisation and not strictly required, we should not fail the entire participation script if gas estimation fails.

Reply:
Right, yes, I will make an update to this.